index.html

```diff
@@ -9,58 +9,66 @@
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet" />
   </head>
 
   <body>
-    <nav class="navbar navbar-expand-lg navbar-light bg-light p-2">
-      <div class="container-fluid">
-        <a class="navbar-brand" href="#">
-          <h2 class="logo">My Logo</h2>
-        </a>
-        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
-        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
-          <div class="navbar-nav">
-            <a class="nav-link" href="#">Portfolio</a>
-            <a class="nav-link" href="#">About</a>
-            <a class="nav-link" href="#">Contact</a>
+    <header>
+      <nav class="navbar navbar-expand-lg navbar-light bg-light p-2">
+        <div class="container-fluid">
+          <a class="navbar-brand" href="#">
+            <h2 class="logo">My Logo</h2>
+          </a>
+          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+            <span class="navbar-toggler-icon"></span>
+          </button>
+          <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+            <div class="navbar-nav">
+              <a class="nav-link" href="#">Portfolio</a>
+              <a class="nav-link" href="#">About</a>
+              <a class="nav-link" href="#">Contact</a>
+            </div>
           </div>
         </div>
-      </div>
-    </nav>
+      </nav>
+    </header>
 
-    <div class="headline">
-      <div class="intro">
-        <h2 style="font-size: clamp(30px, 4vw, 45px); font-weight: 700">
-          I’m Fatihah <br />
-          Glad to see you!
-        </h2>
-      </div>
-      <div class="bio">
-        <p style="font-size: clamp(16px, 4vw, 20px)">
-          I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-        </p>
-      </div>
-      <div class="socials">
-        <h4 class="h6">LET'S CONNECT</h4>
-        <ul>
-          <ol>
-            <a href="#"><img src="./icons/github.png" alt="github icon" /></a>
-          </ol>
-          <ol>
-            <a href="#"><img src="./icons/linkedin.png" alt="linkedin icon" /></a>
-          </ol>
-          <ol>
-            <a href="#"><img src="./icons/twitter.png" alt="twitter icon" /></a>
-          </ol>
-          <ol>
-            <a href="#"><img src="./icons/medium.png" alt="medium icon" /></a>
-          </ol>
-        </ul>
+    <section>
+      <div class="headline">
+        <div class="intro">
```

Comment:
- [Optional] Good job applying the requested changes @Jeddaa, But I would suggest you replace the ` <div class="headline">` with a ` <section class="headline">` instead of wraping it inside another section. This makes your code more organized and simplified.